kas-installer-defaults.env

```diff
@@ -6,12 +6,12 @@ trap 'echo "** Unbound variable must be configured in kas-installer.env or user
 RH_USERNAME="${RH_USERNAME}"
 
 # [required]
-# Owning users RH account_id. See `ocm token | awk -F'.' '{print $2}' | base64 -d | jq -r .account_id`
+# Owning users RH account_id. See `str=$(ocm token | awk -F'.' '{print $2}')===; echo ${str:0:${#str}&-4} | base64 -d | jq -r .account_id`
```

Comment:
I see what this is doing, but I'm not familiar with what `&-4` means. Can you link to any docs on that? Coming up with a good search term isn't going well :)

Reply:
I took this from an answer on https://unix.stackexchange.com/questions/631501/base64-d-decodes-but-says-invalid-input

Looking at it more it would appear to be the bitwise and of -4, which should cause the length to loose the last two bits.

Reply:
Thanks, makes sense. Let me know if you're ready to merge and don't have access.